applications/nrf_desktop/src/modules/cpu_meas.c

```diff
@@ -48,16 +48,6 @@ static void init(void)
 	initialized = true;
 
 	k_work_init_delayable(&cpu_load_read, cpu_load_read_fn);
-	int err = cpu_load_init();
-
-	if (err) {
-		LOG_ERR("CPU load init failed, err: %d", err);
-		module_set_state(MODULE_STATE_ERROR);
-	} else {
-		k_work_reschedule(&cpu_load_read,
-			K_MSEC(CONFIG_DESKTOP_CPU_MEAS_PERIOD));
-		module_set_state(MODULE_STATE_READY);
-	}
```

Comment:
This change breaks the module as the work item (`cpu_load_read`) is not kick-started in the `init` function.

Consider replacing the removed fragment with the following code:

	int ret = cpu_load_get();

	if (ret < 0) {
		LOG_ERR("CPU load init failed, err: %d", ret);
		module_set_state(MODULE_STATE_ERROR);
	} else {
		k_work_reschedule(&cpu_load_read,
			K_MSEC(CONFIG_DESKTOP_CPU_MEAS_PERIOD));
		module_set_state(MODULE_STATE_READY);
	}